twake/backend/node/src/services/user/web/routes.ts

```diff
@@ -50,8 +50,6 @@ const routes: FastifyPluginCallback<{
   fastify.route({
     method: "GET",
     url: "/companies/:id",
-    preHandler: accessControl,
-    preValidation: [fastify.authenticate],
```

Comment:
Why?

Reply:
from the task:
_Get a company by id and public information (this route is public and doesn't need to be authenticated)_

Reply:
Yes, companies general identity is public in Twake php. We can change this later of course (it is used for public shared links like folders or tabs).
We keep it public until we have a concept of "public JWT in-url token" or something.